hopp/simulation/hopp_interface.py

```diff
@@ -42,8 +42,8 @@ def __init__(self, configuration: Union[dict, str, Path]):
         elif isinstance(self.configuration, dict):
             self.hopp = Hopp.from_dict(self.configuration)
 
-    def reinitialize(self):
-        pass
+    def reinitialize(self, configuration: Union[dict, str, Path]):
+        self.__init__(configuration)
```

Comment:
Small nitpick here. It's not unsafe to call `__init__` manually, but it's also not conventional Python. Alternatively you could pull the init logic into `reinitialize` and have `__init__` call `reinitialize`, or could have a separate method like `reset` that is called from both `__init__` and `reinitialize`

Reply:
Thanks, I have now fixed this.